Dockerfile

```diff
@@ -45,6 +46,7 @@ COPY --from=golang_120 /usr/local/go /usr/local/go/go1.20
 COPY --from=golang_121 /usr/local/go /usr/local/go/go1.21
 COPY --from=node /usr/local/lib/node_modules/corepack /usr/local/lib/corepack
 COPY --from=node /usr/local/bin/node /usr/local/bin/node
+COPY --from=ruby /usr/local/bin/ruby /usr/local/bin/ruby
```

Comment:
Copying the executable isn't by far enough, you need shared (.so) libraries for the runtime as well and bundler installation on top of it too. Having said that - do we have a minimum version requirement for ruby/bundler? I don't think so. We have legitimate reasons we copy over stuff for NodeJS and Golang:

- the version of NodeJS shipped by UBI is ~16 and we needed a higher version of corepack, most notably for future yarn v4 support
- we need the latest Go releases due to minimum version requirements in `go.mod` files which many upstream projects that use cachi2 pin to the very latest upstream Go release.

One could argue that Ruby 3.0 (UBI 9 has 3.0.7) has been eol'd in upstream, but we don't really need Ruby (for the core logic, we only need it to run the helper script), we need bundler, so as long as we don't need specific bundler features I'd stay away from copying stuff over from docker images by default and instead install whatever we need the usual way - using the platform package manager -> DNF and only fall back to docker layer copying when that is insufficient.

Reply:
My bad, I should have tested it.